src/lib/server/eliommod_cookies.ml

```diff
@@ -42,11 +50,23 @@ module Persistent_cookies = struct
      the initialization of eliom in case we use static linking with
      sqlite backend ... *)
 
-  type date = float
-  type cookie = Eliom_common.full_state_name * date option * Eliom_common.timeout * Eliom_common.perssessgrp option
+  type stored_cookie = Eliom_common.full_state_name * date option * Eliom_common.timeout * Eliom_common.perssessgrp option
 
   module Ocsipersist = Ocsipersist.Functorial
 
+  (* for backwards compatibility we cannot apply [Marshal] directly to [cookie];
```

Comment:
Are you sure you changed the data representation?

Reply:
I just assumed that I did. But indeed, according to my tests this was not the case. I have removed the transformation code.